tests/test_latentdistributiontest.py

```diff
@@ -8,96 +8,64 @@
 from sklearn.metrics import pairwise_distances
 
 from graspologic.embed import AdjacencySpectralEmbed
-from graspologic.inference import LatentDistributionTest
+from graspologic.inference import latent_distribution_test
 from graspologic.simulations import er_np, sbm
 
 
 class TestLatentDistributionTest(unittest.TestCase):
     @classmethod
-    def setUpClass(cls):
+    def test_bad_kwargs(self):
         np.random.seed(123456)
-        cls.tests = {"dcorr": "euclidean", "hsic": "gaussian", "mgc": "euclidean"}
-        cls.A1 = er_np(20, 0.3)
-        cls.A2 = er_np(20, 0.3)
-
-    def test_fit_ase_works(self):
-        for test in self.tests.keys():
-            ldt = LatentDistributionTest(test, self.tests[test], n_bootstraps=10)
-            assert ldt.fit(self.A1, self.A2) is ldt
-
-    def test_fit_predict_ase_works(self):
```

Comment:
why did you remove all these tests?

Reply:
I wasn't sure how to make the tests that had "fit" or "fit_predict" in them because it seemed to me that those were tests that could only be made when LPT and LDT were classes rather than functions

Reply:
if there is a test for both fit and fit_predict - only do one. the important part is to make sure that it runs in the setting.

Reply:
eg you no longer seem to have test for omni in lpt

Reply:
Addressed in lines 17-23